extensions/html-language-features/server/src/test/completions.test.ts

```
@@ -64,7 +64,7 @@ export async function testCompletionFor(value: string, expected: { count?: numbe
 
 	const list = await mode.doComplete!(document, position, context);
 
-	if (expected.count) {
+	if (expected.count !== undefined) {
```

Prevent silent errors when testing for `count: 0`